### ISSUE TITLE
Share Post to Facebook 

### ISSUE THREAD

**Issue by whes1015** [adw0rd/instagrapi#882]
### Compensate with donations ($20 and more) for the time I spent to implement new functionality or send me your [Pull Request](https://github.com/adw0rd/instagrapi/pulls) with your implementation of the new functionality

[![Donate](https://www.buymeacoffee.com/assets/img/custom_images/yellow_img.png)](https://www.buymeacoffee.com/adw0rd)

* Tron TRX/USDT: `TLprxuzHjXPkHYa7ZfTTt165p7Fphwy1Ee`
* Bitcoin: `bc1qmp7538hktvg2gx994hrpkmsmyjc03q29awe9ra`
* Ethereum: `0x0E674815d513d4a00F6a7Cd4A990807aB97cC70D`
* Dogecoin: `DSEXzMBYhBTwhNqzPGxVWFsTqVsTCeDejo`

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by subzeroid** [comment]
Keeping this open as a separate feed-post feature request.

Current instagrapi supports Facebook cross-posting for Reels/Clips through `clip_upload(..., share_to_facebook=True)` and the `clip_share_to_fb_*` helpers. That flow is documented here: https://subzeroid.github.io/instagrapi/usage-guide/media.html

Regular feed photo/video post sharing is not the same payload as the modern Reel cross-posting flow. The old raw `extra_data={"share_to_facebook": 1}` flag may still be passed through to low-level configure calls, but it is not a confirmed high-level feed-post Facebook sharing implementation. To add this safely, we need a sanitized current Android app capture of the final feed photo/video configure request with Facebook sharing enabled: endpoint, method, body fields, and response shape.

Assigned/backlogged for a future implementation instead of closing as already solved by the Reels API.

**Comment by solo-entrepreneurs** [comment]
@subzeroid is this closed too?

**Comment by subzeroid** [comment]
Yes, this older request is now resolved by #2750 and was released in [2.18.11](https://github.com/subzeroid/instagrapi/releases/tag/2.18.11).

`photo_upload`, `video_upload`, and `album_upload` now support `share_to_facebook=True`, alongside the existing Reel support in `clip_upload`. The implementation includes linked Facebook destination discovery and explicit `fb_destination_id` / `fb_destination_type` overrides when needed.

Thank you @solo-entrepreneurs for pointing out this older backlog item and for the detailed Facebook cross-posting reports that helped us implement it.
